processor/transformprocessor/config.go

```diff
@@ -0,0 +1,27 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transformprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor"
+
+import (
+	"go.opentelemetry.io/collector/config"
+)
+
+type Config struct {
```

Comment:
Continuing discussion from

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6985#discussion_r777653778

```
Should we have a "Validate" maybe to ensure the query are parsable?
```

_Originally posted by @bogdandrutu in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6985#discussion_r777653778_

Reply:
Thanks for clarifying the difference with Validate, this makes sense

Reply:
If there is anywhere you thing we can update comments to explain the difference would be good.